packages/datadog-plugin-pg/src/index.js

```diff
@@ -36,8 +38,14 @@ function createWrapQuery (tracer, config) {
 
       const originalCallback = pgQuery.callback
       const statement = pgQuery.text
+      const values = pgQuery.values
 
       span.setTag('resource.name', statement)
+      span.setTag('sql.query', statement)
+
+      if (config.includeQueryParams) {
```

Comment:
It would be more flexible if this was a function that receives/returns parameters instead of a boolean. Then it would be possible to pick which params to include/exclude.

Reply:
Params are an array of values so there's no way to distinguish them. It might still be useful to filter them so I'll provide both options.

Reply:
This is done BTW.